### ISSUE TITLE
Slider: aria-valuetext should be customizable

### ISSUE THREAD

**Issue by fbasso** [AmadeusITGroup/AgnosUI#213]
At the moment, it is fixed to (angular version):

```typescript
[attr.aria-valuetext]="item.value"
```
It should be customizable, as in the rating for example.
